BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Service/CellFormatter/TextFormatter.js

```diff
@@ -0,0 +1,18 @@
+import AbstractCellFormatter from './AbstractCellFormatter'
+import TextDataFormatter     from '../DataFormatter/TextFormatter'
+
+/**
+ * @class TextFormatter
+ */
+class TextFormatter extends AbstractCellFormatter
```

Comment:
why this class i dont understand. Why not use directly DataFormatter/TextFormatter